docs/essentials/controls.md

```diff
@@ -31,7 +31,49 @@ If you have stories in the older pre-Storybook 6 style, check the [args & contro
 
 <IfRenderer renderer='angular'>
 
-By default, Storybook will choose a control for each arg based on its initial value. This will work well with specific arg types (e.g., `boolean` or `string`). To enable them, add the `component` annotation to the default export of your story file, and it will be used to infer the controls and auto-generate the matching [`argTypes`](../api/arg-types.md) for your component using [Compodoc](https://compodoc.app/) if you opt-in to use it, including first-class support for Angular's `inputs`, `outputs`, `properties`, `methods` and `view/content child/children`.
+By default, Storybook will try to infer the required argTypes and associated controls for your stories based on the component's definition and initial value of the args using [Compodoc](https://compodoc.app/), a documentation generator for Angular applications that can extract the metadata of your components, including first-class support for Angular's `inputs`, `outputs`, `properties`, `methods`, and `view/content child/children`. If you opt-in to use it, you must take additional steps to set it up properly.
+
+Run the following command to install the tooling.
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+   'angular/compodoc-install.yarn.js.mdx',
+   'angular/compodoc-install.npm.js.mdx',
+   'angular/compodoc-install.pnpm.js.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+Update your `angular.json` file to include the following configuration to include it in the Storybook's inbuilt builder configuration.
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+   'angular/angular-project-compodoc-config.json.mdx',
+  ]}
+/>
+
+
+<!-- prettier-ignore-end -->
+
+Finally, update your `.storybook/preview.ts` file to include the following configuration to import the metadata generated by Compodoc and use it to generate the controls and argTypes for your stories.
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+   'angular/storybook-preview-compodoc-config.ts.mdx',
+  ]}
+/>
+
+
+<!-- prettier-ignore-end -->
+
+When you set the `component` annotation to the default export of your story file, it will be used to infer the controls and auto-generate the matching [`argTypes`](../api/arg-types.md) for your component.
```

Comment:
This tiny change is actually quite important, as it significantly corrects the meaning of the sentence.

```suggestion
When you set the `component` annotation of the default export of your story file, it will be used to infer the controls and auto-generate the matching [`argTypes`](../api/arg-types.md) for your component.
```